### PR TITLE
Store flight statistics in log file and offer file explorer for flight statistics, v2

### PR DESCRIPTION
**Pull request by bubeck**
Added a "Flight log" in the settings menu to select
a previous log file and see the flight statistics again.
This is used as an extension point to store even more
valuable data inside the log file and present it to
the pilot after the flight.

Fixes the following requests:
#208
#73

### CODE REVIEW

File: skydrop/src/fc/logger/igc.cpp [hunk 1]
```diff
@@ -218,5 +213,6 @@ void igc_comment(char * text)
 
 void igc_stop()
 {
+	igc_write_grecord();
```
Comment (fhorinek) [inline skydrop/src/fc/logger/igc.cpp]: The reason for writing G-record after every fix is that pilot will allays get valid record even when power suddenly drop mid flight. 

We should call igc_write_grecord(); after every added line, or comment

File: skydrop/src/gui/settings/settings.cpp [hunk 2]
```diff
@@ -52,6 +52,9 @@ void gui_settings_action(uint8_t index)
 	case(6):
 		gui_switch_task(GUI_SET_DEBUG);
 	break;
+	case(7):
```
Comment (fhorinek) [inline skydrop/src/gui/settings/settings.cpp]: Put it just under logger menu entry

File: skydrop/src/fc/fc.cpp [hunk 3]
```diff
@@ -416,12 +416,19 @@ void fc_landing()
...
+	audio_off();
+
+
 	fc.flight.state = FLIGHT_LAND;
 	fc.flight.autostart_timer = task_get_ms_tick();
-
 	fc.flight.timer = task_get_ms_tick() - fc.flight.timer;
 
-	audio_off();
+	logger_comment(PSTR(" SKYDROP-START-s: %lu "), time_get_local());
```
Comment (fhorinek) [inline skydrop/src/fc/fc.cpp]: why not use flight duration to get real start time?
time_get_local() - (fc.flight.timer / 1000)

File: skydrop/src/gui/settings/settings.cpp (hunk 2 above)
Comment (bubeck) [inline skydrop/src/gui/settings/settings.cpp]: done

File: skydrop/src/fc/fc.cpp (hunk 3 above)
Comment (bubeck) [inline skydrop/src/fc/fc.cpp]: done

File: skydrop/src/fc/logger/igc.cpp (hunk 1 above)
Comment (bubeck) [inline skydrop/src/fc/logger/igc.cpp]: done

**Comment by fhorinek** [comment]
Thank you, I will test it tomorow

**Comment by bubeck** [comment]
Yes, me too. I am currently in Col Rodella in the Dolomites for skiing and flying. VERY funy. In the evening I do some coding and test the other day. :)

**Pull request merged**